### PR TITLE
Improvement/new endpoints and cors

### PR DESCRIPTION
**Pull request by YashSahsani**
1)Changed the request type and added new endpoints 
2) Added cors (Cross-Origin Resource Sharing)

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -5,11 +5,22 @@
...
 from pydantic import BaseModel
+from fastapi.middleware.cors import CORSMiddleware
 app = FastAPI()
+origins = [
+    "https://youtubenlp.com",
+    "http://localhost",
+    "https://localhost",
+    "http://localhost:8080",
+    "https://localhost:8080",
+]
```
Comment (YashSahsani) [inline main.py:16]: This right now doesn't matter as i have allowed allow_orgins

**Comment by YashSahsani** [comment]
youtube

File: main.py [hunk 2]
```diff
@@ -5,23 +5,41 @@
...
+def transcripts(video_id: str):
 	transcript_list = jsonable_encoder(get_transcripts(video_id))
 	return JSONResponse(content=transcript_list)
 
-@app.post('/score/comments')
-def comments(item : Item):
-	video_id = item.videoID
-	kTop = item.kTopComments
-	comments_list = get_comments(video_id, kTop)
+@app.get('/score/comments/{video_id}/{kTopComments}')
```
Comment (pandevim) [inline main.py:30]: If you see the current [API Spec](https://app.swaggerhub.com/apis-docs/youtubenlp/backend/0.0.1) there is no `kTopComments` parameter right now. So may dropping it won't harm us. We have to implement this in future. Sorry for the confusion.

Comment (pandevim) [reply main.py:30]: Same changes should be reflected in the README.md

**Pull request closed without merging**